tests/framework/results/test_context.py

```diff
@@ -74,6 +78,11 @@ def _aggregate_state_person_time(self, x: pd.DataFrame) -> float:
     return len(x) * (28 / 365.35)
 
 
+def _aggregate_state_person_time_function(x: pd.DataFrame) -> float:
```

Comment:
`_aggregate_state_person_time` and `_aggregate_state_person_time_function` are identical. Is one of them supposed to be returning a callable?

Reply:
Good catch, the difference was there for mocking into an object (the `self` param), but there's no apparent reason for that. I'll refactor that out!